sdk/metricsadvisor/ai-metrics-advisor/review/ai-metrics-advisor.api.md

```diff
@@ -204,6 +204,7 @@ export type DataFeed = {
     creator: string;
     source: DataFeedSource;
     schema: DataFeedSchema;
+    metricIds: DataFeedMetricMap;
```

Comment:
I don't think we need an alias type here. Let's just use 

```suggestion
    metricIds: Map<string, string>;
```

Reply:
included suggestions in latest commit